weblate/trans/tests/test_component.py

```diff
@@ -733,6 +748,13 @@ def test_no_matches(self):
             self.component.full_clean,
         )
 
+        self.component.screenshot_filemask = "foo/*.png"
+        self.assertRaisesMessage(
+            ValidationError,
+            "The file mask did not match any files.",
+            self.component.full_clean,
+        )
```

Comment:
Please create a separate test here. In this case, you have no clue whether it is validation from `filemask` or `screenshot_filemask`.

Reply:
Since we are not performing a full repo scan on screenshot file mask change, imo, we can just check for valid/invalid ss filemask and there is no need to check if there are file matching that mask in the repo. wdyt?